homeassistant/components/blue_current/const.py

```diff
@@ -8,6 +8,12 @@
 
 EVSE_ID = "evse_id"
 MODEL_TYPE = "model_type"
+CARD = "card"
+UID = "uid"
+BCU_APP = "BCU-APP"
+WITHOUT_CHARGING_CARD = "without_charging_card"
+CHARGING_CARD_ID = "charging_card_id"
+START_CHARGE_SESSION = "start_charge_session"
```

Comment:
```suggestion
SERVICE_START_CHARGE_SESSION = "start_charge_session"
```
Maybe clearer to recognize